src/wasm-lib/kcl/src/parser.rs

```diff
@@ -183,7 +211,7 @@ impl Parser {
 
     /// Use the new Winnow parser.
     pub fn ast(&self) -> Result<Program, KclError> {
-        parser_impl::run_parser(&mut self.tokens.as_slice())
+        parser_impl::run_parser(&mut self.tokens.as_slice(), &mut self.unkown_tokens.as_slice())
```

Comment:
Let's validate that `unkown_tokens` (should be `unknown_tokens`) is empty here, in `Parser::ast` instead of in `run_parser` below.